composer.json

```diff
@@ -1,48 +1,57 @@
 {
-  "name": "silverleague/silverstripe-ideannotator",
-  "description": "Generate PHP DocBlock annotations for DataObject and DataExtension databasefields and relation methods",
-  "type": "silverstripe-vendormodule",
-  "keywords": [
-    "silverstripe",
-    "database",
-    "orm",
-    "docblock",
-    "ide",
-    "annotations"
-  ],
-  "license": "BSD-3-Clause",
-  "authors": [
-    {
-      "name": "Martijn van Nieuwenhoven",
-      "email": "info@axyrmedia.nl"
+    "name": "silverleague/ideannotator",
+    "description": "Generate PHP DocBlock annotations for DataObject and DataExtension databasefields and relation methods",
+    "type": "silverstripe-vendormodule",
+    "keywords": [
+        "silverstripe",
+        "database",
+        "orm",
+        "docblock",
+        "ide",
+        "annotations"
+    ],
+    "license": "BSD-3-Clause",
+    "authors": [
+        {
+            "name": "Martijn van Nieuwenhoven",
+            "email": "info@axyrmedia.nl"
+        },
+        {
+            "name": "Simon 'Firesphere' Erkelens",
+            "email": "simonerkelens@silverstripe.com"
+        }
+    ],
+    "require": {
+        "php": ">=5.6.0",
+        "silverstripe/recipe-cms": "1.0.x-dev",
+        "phpdocumentor/reflection-docblock": "^2.0@dev"
     },
-    {
-      "name": "Simon 'Firesphere' Erkelens",
-      "email": "simonerkelens@silverstripe.com"
-    }
-  ],
-  "require": {
-    "php": ">=5.6.0",
-    "silverstripe/recipe-plugin": ">=0.1@dev <1.0",
-    "silverstripe/recipe-cms": "1.0.x-dev",
-    "phpdocumentor/reflection-docblock": "^2.0@dev"
-  },
-  "require-dev": {
-    "scriptfusion/phpunit-immediate-exception-printer": "^1",
-    "phpunit/phpunit": "^5.7"
-  },
-  "autoload": {
-    "psr-4": {
-      "Axyr\\IDEAnnotator\\": "src/"
-    }
-  },
-  "extra": {
-    "branch-alias": {
-      "3.x-dev": "3.0.x-dev",
-      "dev-master": "3.x-dev"
+    "require-dev": {
+        "scriptfusion/phpunit-immediate-exception-printer": "^2",
+        "phpunit/phpunit": "^5.7"
     },
-    "installer-name": "ideannotator"
-  },
-  "prefer-stable": true,
-  "minimum-stability": "dev"
+    "autoload-dev": {
+        "psr-4": {
+            "SilverLeague\\IDEAnnotator\\": "src/"
+        }
+    },
+    "extra": {
+        "branch-alias": {
+            "3.x-dev": "3.0.x-dev",
+            "dev-master": "3.x-dev"
+        },
+        "installer-name": "ideannotator",
+        "project-files-installed": [
+            ".htaccess",
+            "index.php",
+            "install-frameworkmissing.html",
+            "install.php",
+            "mysite/.htaccess",
+            "mysite/_config.php",
+            "mysite/_config/mysite.yml",
+            "web.config"
+        ]
```

Comment:
This configuration is for recipes - don't think we need this. Users might have problems using this with silverstripe/recipe-cms perhaps - thoughts?